src/test/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfigTest.java

```diff
@@ -0,0 +1,49 @@
+package io.confluent.connect.elasticsearch;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ElasticsearchSinkConnectorConfigTest {
+
+  private Map<String, String> createProps() {
```

Comment:
Where we have used this pattern of manually calling `createProps` and `setUp` in other connectors without using the `@Before` junit annotation, we've done so because some properties in only a few test cases need to be set/customized before the `setUp` method is called. 

If there's no such requirement in a test class, I'd suggest we stick with the standard way of using the `@Before` annotation for initializations.